src/routes/viewer/ModelView.svelte

```diff
@@ -35,7 +36,12 @@
 			console.log('Model', model);
 			mesh = model as THREE.Mesh;
 		}
-		let bbox = new THREE.Box3().setFromObject(model);
+		let thisbbox = new THREE.Box3().setFromObject(model);
+        if (bbox === null) {
+            bbox = thisbbox.clone();
```

Comment:
I'm not 100% sure, but I *think* that this will not pass upward.  What I mean is that I think you can modify mutable objects that get passed in, but it may be the case that this won't go back up to the `Scene` if `Scene` passes in a null/undefined.

(Mind you, I'm saying this, and it's something that somebody more experience with Svelte could just say "oh yeah totally" or "no that's not how it works" :) )